lib/ramble/ramble/cmd/workspace.py

```diff
@@ -1446,6 +1446,134 @@ def workspace_manage_includes(args):
             ws.add_include(args.add_include)
 
 
+def workspace_manage_modifiers_setup_parser(subparser):
+    """manage workspace modifiers"""
+    actions = subparser.add_mutually_exclusive_group()
+    actions.add_argument(
+        "--list", "-l", action="store_true", help="whether to print existing modifiers"
+    )
+
+    actions.add_argument(
+        "--add",
```

Comment:
what happens if none of add, remove or list are passed?

Reply:
We can make arg parse request one of the three is set by using a mutual exclusive group.. would that be useful? 

Might also be useful to think about what should happen is someone asks for both in the same invocation

Reply:
This is already using a mutually exclusive group. Maybe I need to add something to say that the default is list.